### ISSUE TITLE
Support alias for listProperty options

### ISSUE THREAD

**Issue by emersonsiega** [erickzanardo/dashbook#53]
Hi

I'm working on a component that receives a prefix/suffix widget. I was trying to add a short set of example widgets to show how this parameter affects the component.

The problem is the options doesn't have an alias, so they are rendered like this:
![image](https://user-images.githubusercontent.com/2227499/123158540-4d047580-d442-11eb-9f11-1665dda077b8.png)

```dart
prefix: context.listProperty(
        'prefix',
        null,
        [
          null,
          Container(
            padding: const EdgeInsets.only(right: 8),
            child:  Icon(Icons.pin_drop, size: 16),
          ),
          Container(
            padding: const EdgeInsets.only(right: 8),
            child:  Icon(Icons.check, size: 16),
          ),
        ],
      ),
```

I was thinking that a simple alias can resolve this issue.

(Just an example): At the storybook_flutter package there's an `Option` class, where you can set an alias for each item.
_(I've just migrated from this package due to some UI problems that was bothering me)_

**Comment by erickzanardo** [comment]
Thanks for the suggestions, I have been thinking on having some sort of `Option` class as well, cause sometimes the `toString` is not enough for a good preview.

I think I will keep `listProperty` as is, because for simple values, it works very nice, and will probably add a new property, an `optionsProperties` maybe. What do you think, would that solve the issue?

**Comment by emersonsiega** [comment]
Thanks for the fast answer :)

Yeah, sure! 

There's some situations that the `toString` works well, so it'll be nice to keep a "simpler" way to declare the options.

**Comment by erickzanardo** [comment]
Great!

I was planning on working on a couple of features for Dashbook this week, so I will include this on my TODO list, should be simple enough to make.

**Comment by erickzanardo** [comment]
@emersonsiega I just submitted #53, I have added a preview video on the PR description, let me know if that would work for your need.

**Comment by emersonsiega** [comment]
Really nice! 

This will help me a lot! Thank you very much :D

**Comment by erickzanardo** [comment]
Shipped on `0.1.3`
